tests/suites/main_test.function

```diff
@@ -467,10 +470,21 @@ int main(int argc, const char *argv[])
 
                 unmet_dep_count = 0;
             }
-            else if( ret == DISPATCH_TEST_SUCCESS && test_errors == 0 )
+            else if( ret == DISPATCH_TEST_SUCCESS )
             {
-                mbedtls_fprintf( stdout, "PASS\n" );
-                fflush( stdout );
+                if( test_info.failed == 0 )
+                {
+                    mbedtls_fprintf( stdout, "PASS\n" );
+                    fflush( stdout );
+                }
+                else
+                {
+                    total_errors++;
+                    mbedtls_fprintf( stdout, "FAILED\n" );
+                    mbedtls_fprintf( stdout, "  %s\n  at line %d, %s\n",
+                                     test_info.test, test_info.line_no,
+                                     test_info.filename );
+                }
```

Comment:
Would it not be better to have a `fflush( stdout );` between lines 487 and 488 rather than at 478?

If not, shouldn't we have one after line 486?

Reply:
Good point! Thanks, just did it.